ui/addon/src/main/java/org/jboss/windup/ui/WindupUpdateRulesetCommand.java

```diff
@@ -62,6 +62,11 @@ public Result execute(UIExecutionContext context) throws Exception
         Coordinate latestCoordinate;
         do{
             i++;
+            i++;
```

Comment:
Also, a few lines above, the fail message is: "The ruleset is already in the most updated version."

This should be changed to Results.success, with a message of: "The ruleset is up to date!"